### PR TITLE
Task/10.1 recuperar borrados

### PR DESCRIPTION
**Pull request by Ral15**
#### What's this PR do?
Creates a button for restoring estudios
#### Where should the reviewer start?
dashboard.hbs

#### How should this be manually tested?
Create an estudio, delete it, and then check the section of eliminados

#### Any background context you want to provide?

This template was adapted ~retrived~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: test/restoreEstudio.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+global.before(function () {
+    chai.should();
+    chai.use(chaiAsPromised);
+});
+
+
+describe('Create Estudio test', function () {
+  /**
+  * Integration test suite for testing the Login.
```
Comment (erikiado) [inline test/restoreEstudio.js:38]: This is not testing the login

File: test/restoreEstudio.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  //     .then(async () => {
+  //       await sleep(1000);
+  //       return client.getText('#modalContentId');
+  //     })
+  //     .then((modalText) => {
+  //       assert.equal(modalText, 'No podras recuperar el estudio después de esta acción.');
+  //     });
+  // });
+  // /**
+  // * Test delete estudio
```
Comment (erikiado) [inline test/restoreEstudio.js:142]: What is this?

File: test/restoreEstudio.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  //       await sleep(1000);
+  //       return client.click('#show-deleted');
+  //     })
+  //     .then(async () => {
+  //       await sleep(1000);
+  //       return client.isVisible('#delete-estudio-' + estudioId);
+  //     })
+  //     .then((isVisible) => {
+  //       assert.isTrue(isVisible);
+  //     })
```
Comment (adael2) [inline test/restoreEstudio.js:194]: This commented test is not a part of the task or why is commented?

File: test/restoreEstudio.js (hunk 2 above)
Comment (Ral15) [reply test/restoreEstudio.js:142]: @erikiado a comment of a test that will no be used, if you look at all the PR you will see that it is used later.

File: test/restoreEstudio.js (hunk 3 above)
Comment (Ral15) [reply test/restoreEstudio.js:194]: @adal347  check all the PR's.

**Review by adael2 — APPROVED** [review]

**Review by erikiado — APPROVED** [review]

**Pull request closed without merging**